enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/SecurityLog.java

```diff
@@ -38,35 +39,31 @@
 import static org.neo4j.helpers.Strings.escape;
 import static org.neo4j.kernel.impl.enterprise.configuration.EnterpriseEditionSettings.security_log_filename;
 
-public class SecurityLog implements Log
+public class SecurityLog implements Log, Lifecycle
 {
+    private RotatingFileOutputStreamSupplier rotatingSupplier;
     private final Log inner;
 
     public SecurityLog( Config config, FileSystemAbstraction fileSystem, Executor executor ) throws IOException
     {
-        this( createLog( config, fileSystem, executor ) );
-    }
-
-    public SecurityLog( Log log )
-    {
-        inner = log;
-    }
```

Comment:
Sure, I mostly meant that it wouldn't compile without it. But I see now that you have added it again below.